pkg/test/resourcefixture/testdata/basic/monitoring/v1beta1/monitoringdashboard/monitoringdashboardfull/_generated_export_monitoringdashboardfull.golden

```diff
@@ -3,11 +3,13 @@ kind: MonitoringDashboard
 metadata:
   name: monitoringdashboard-${uniqueId}
 spec:
-  columnLayout:
```

Comment:
Maybe I miss something, do we want to remove this field? I still see the spec.columnlayout in the generated doc.

Reply:
Good point, was cleaned up in a rebase.  Technically these aren't actually allowed in a columnLayout on GCP, so I will eventually switch the tests so that it passes all the GCP validation (there's a lot of it!)